docker-init-script/src/main/java/eu/xenit/docker/alfresco/InitScriptMain.java

```diff
@@ -131,9 +131,13 @@ public void process() throws IOException {
         if (customKeystores) {
             try (InputStream keystorePropertiesInputStream = new FileInputStream(dirKeystore.concat("/ssl-keystore-passwords.properties"))) {
                 keystoreProperties.load(keystorePropertiesInputStream);
+            } catch (IOException ioException) {
+                System.err.println("[ERROR] failed to load custom ssl-keystore-passwords.properties file at location");
```

Comment:
Let it throw an exception instead of printing an error.
The init-script should exit unsuccesfully when this happens, not log some error and happily continue with a bad configuration.